pkgs/games/npush/run.nix

```diff
@@ -0,0 +1,39 @@
+{ runtimeShell
+, symlinkJoin
+, writeTextFile
+, npush
+}:
+
+let
+  runScript = writeTextFile {
```

Comment:
Could this maybe use `writeShellScriptBin`?

Reply:
Yes, but I think `writeTextFile` is more clear. I always forget the meaning and position of the variables...